Makefile

```diff
@@ -56,28 +54,21 @@ $(VIM_PATH_BUILD).utf-8.spl: $(HUNSPELL_PATH_BUILD) | $(BUILD_DIR)
 # * "Wi-fi" -> "Wi" + "fi".
 # Apparently, it also can't handle UTF-8. Wonderful language software.
 
-# FIXME: I should not have to apply the same regex twice to resolve e.g.
-# "Cortex-A53" -> "Cortex" + nothing else
-.INTERMEDIATE: $(ASPELL_PATH_BUILD).tmp
-$(ASPELL_PATH_BUILD).tmp: $(HUNSPELL_PATH_BUILD)
-	cat $< | \
-	  iconv -f 'UTF-8' -t 'ASCII//TRANSLIT' | \
-	  sed -r 's/([-_.,:;!?#@$%^&*~+0-9]+)([^\W\d_]*)/\n\2/g' | \
-	  sed -r 's/([-_.,:;!?#@$%^&*~+0-9]+)([^\W\d_]*)/\n\2/g' | \
-	  sed -r "s/^'.*//g" | \
-	  sed -n '/../ p' | \
-	  sort -u \
-	  > $@
-
+$(ASPELL_PATH_BUILD): $(AFFIX_BUILD) | $(BUILD_DIR)
+	cat $<                                | \
+	sed 's/\W/\n/g;s/_/\n/g;s/[0-9]//g;'  | \
```

Comment:
There is an issue here: you are replacing [0-9] with nothing, hence e.g.: "Cortex-A56A" would turned into:

```
Cortex
AA
```

when it should be turned into:

```
Cortex
A
A
```

Suggestion with -r that also avoids using multiple regexes:
`sed -r 's/(\W|_|[0-9])+/\n/g'`

(Or without -r: `sed 's/\(\W|_|[0-9]\)+/\n/g'`)